server/src/main/resources/application.properties

```diff
@@ -0,0 +1,27 @@
+
+spring.datasource.url= jdbc:mysql://localhost:3306/PMS
+spring.datasource.username=root
+spring.datasource.password=rootroot
+spring.jpa.hibernate.ddl-auto=none
+spring.jpa.database-platform=org.hibernate.dialect.MySQL5Dialect
+spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.MySQL5InnoDBDialect
+spring.jpa.show-sql=true
+jwt.secret=PMS
```

Comment:
@SunilSunny698 Have an min 32 bit value as an secret key ( its more like password to your application, weak secret mean weak security no matter how good you have configured security system.)